src/Helpers/Filters/Filters.jsx

```diff
@@ -0,0 +1,78 @@
+export const getSortedProducts =  (products, sortBy) =>{
+    if(sortBy === ''){
+        return products
+    }
+    // console.log("prod..products",products );
+    if(sortBy ==='LOW_TO_HIGH'){
+        //SORT ACC TO DISCOUNTED PRICE
+        return [...products].sort((item1, item2)=> 
+            (item1.mrp-(item1.mrp * (item1.discount/100))) - (item2.mrp-(item2.mrp * (item2.discount/100)))
+        )
```

Comment:
Suggestion: You can precompute this value in a variable (item1.mrp * (item1.discount/100)), it would increase the readability

Reply:
ok